### ISSUE TITLE
Text Contrast Issue

### ISSUE THREAD

**Issue by Gurutejareddy9** [dayashankar-ai/interactive-deep-learning-lectures#1]
### 🐛 Bug Description
There are two separate issues related to content visibility. In Lecture 7, the visualization bars for the gradient descent methods are a similar color to the background, making them hard to see. In Lecture 8, the text within a code block has a light font on a light background, making it unreadable.

***

### 📍 Location
- **Lecture:** Lecture 7: Mini-Batch Stochastic Gradient Descent and Lecture 8: Advanced Optimizers
- **Section:** "Interactive Understanding" (Lecture 7) and "Hyperparameter Tuning Strategy" (Lecture 8)
- **URL:** - `https://dayashankar-ai.github.io/interactive-deep-learning-lectures/lectures/lecture07-mini-batch-stochastic-gradient-descent.html`
  - `https://dayashankar-ai.github.io/interactive-deep-learning-lectures/lectures/lecture08-advanced-optimizers.html`

***

### 🔄 Steps to Reproduce
1. Go to the Lecture 7 URL.
2. Scroll to the "Visualization Exercise" section.
3. Observe the visualization bars, which are barely visible against the white background.
4. Go to the Lecture 8 URL.
5. Scroll down to the "Hyperparameter Tuning Strategy" section.
6. The code block text is unreadable due to a lack of color contrast.

***

### ✅ Expected Behavior
- For Lecture 7, the visualization bars should have a clear, contrasting color so they are easily visible.
- For Lecture 8, the code block text should be a dark color (e.g., black) to be readable on the white background.

***

### ❌ Actual Behavior
- The visualization bars in Lecture 7 are a light gray, blending into the white background.
- The code text in Lecture 8 is a white color on a white background, rendering it invisible.

***

### 📱 Environment
- **Device:** Desktop
- **Operating System:** macOS
- **Browser:** Chrome
- **Browser Version:** 127.0.6533.72
- **Screen Size:** N/A

***

### 📸 Screenshots
- **Lecture 7:** 

<img width="1128" height="794" alt="Image" src="https://github.com/user-attachments/assets/68c427a5-7fb0-4ea2-bfc2-0630313cf340" />

- **Lecture 8:** 

<img width="1161" height="794" alt="Image" src="https://github.com/user-attachments/assets/89a52d43-1106-4d80-a3e0-4bc0ab53ac09" />

***

### 🎓 Impact on Learning
- [ ] Prevents completion of lecture
- [X] Interactive demo not working
- [ ] Quiz/assessment issues
- [ ] Navigation problems
- [X] Content display issues
- [ ] Other: The bug hinders the core learning experience by making critical visual aids and code examples unviewable.

***

### 📝 Additional Context
N/A

***

### 🔧 Possible Solution
- **Lecture 7:** Adjust the CSS for the visualization bars to use a color with a higher contrast against the background.
- **Lecture 8:** Change the font color within the code block to a dark color to ensure readability against the light background.
